src/main/java/au/edu/wehi/idsv/configuration/VariantCallingConfiguration.java

```diff
@@ -116,6 +117,10 @@ public VariantCallingConfiguration(Configuration config) {
 	 * Include single_assembly filtering in output vcf
 	 */
 	public boolean includeSingleAssemblyFilter;
+	/**
+	 * Minimum number of bases that consider overlap between read and assembly to be considered for assembly support
```

Comment:
I think the description (and the parameter name) is a bit confusing. We do not require high overlap between the read and the assembly, but high overlap between the read and the breakpoint that the assembly generates. 
Can you try finding a different name and description?